configs/train.cfg

```diff
@@ -14,12 +14,29 @@ experiment.triplet_margin = 0.1
 experiment.hits_normalizer = @ConstraintsNormalizer()
 experiment.num_workers = 4
 experiment.pin_memory = True
-#experiment.resume_from_checkpoint = 
+
+experiment.metrics = [
+    @SilhouetteScoreMetric(),
+    @DaviesBouldinScoreMetric(),
+    @CalinskiHarabaszScoreMetric(),
+    @AccuracyScoreMetric(),
+    @F1ScoreMetric(),
+    @PrecisionScoreMetric(),
+    @RecallScoreMetric(a)
+]
+#experiment.resume_from_checkpoint =
 
 ### model ###
 TrackEmbedder.n_blocks = 2
 TrackEmbedder.output_dim = 32
 
+### clustering model ###
+clustering_algorithm.module = 'sklearn.cluster'
```

Comment:
I am not sure, but maybe it is better to pass it explicitly to experiment arguments